Multitrack-Template.lua

```diff
@@ -0,0 +1,205 @@
+--[[
+Multi-track re-record script template.
+
+This template was created from DarkKobold's Multitrack LUA script created
+for TAS of TMNT Arcade and The Simpsons Arcade.
+
+The intent of this template script is to make multiplayer TAS'ing more
+approachable to newbies - empowering them to quickly create a
+multi-track re-recording rig for the game they plan to TAS
+--]]
+
+-- Max number of lines to display in LUA overlay
+MAX_DISPLAY_LINES = 5;
+
+-- local NameTable = {'Player 1', 'Player 2', 'None', 'All'};
+local NameTable = {'Player 1', 'Player 2'};
```

Comment:
As you can see from this template WIP [currently filled in with double dragon values]. I, for the most part, only have to tweak values at the top of the script as I make directions, new players, keypad entries, etc. available for my TAS